spec/jobs/import_from_vacancy_sources_job_spec.rb

```diff
@@ -133,7 +133,7 @@ def each(...)
           "slug" => "mallowpond-high-school",
           "start_date_type" => "specific_date",
           "starts_asap" => nil,
-          "starts_on" => "2023-12-06",
+          "starts_on" => Date.today,
```

Comment:
We're changing string to date here inside `eq`. How is it not breaking the test?